packages/account/src/Components/api-token/__tests__/api-token-clipboard.spec.tsx

```diff
@@ -8,7 +8,7 @@ modal_root_el.setAttribute('id', 'modal_root');
 document.body.appendChild(modal_root_el);
 
 describe('ApiTokenClipboard', () => {
-    const mock_props = {
+    const mock_props: React.ComponentProps<typeof ApiTokenClipboard> = {
```

Comment:
Can we use a direct export here?

Reply:
@sanjam-deriv , why?
Isn’t it better way to extract component types rather than export them?
if types are not supposed to be reused and to be extended, we better should use this approach I believe. need other opinions on this :)

Reply:
++ @yauheni-deriv